runtime/autoload/ccomplete.lua

```diff
@@ -24,10 +24,12 @@ local SearchMembers = nil
 -- vim9script
 
 -- # Vim completion script
--- # Language:     C
--- # Maintainer:   Bram Moolenaar <Bram@vim.org>
+-- # Language:	C
+-- # Maintainer:	The Vim Project <https://github.com/vim/vim>
+-- # Last Change:	2023 Aug 10
 -- #		Rewritten in Vim9 script by github user lacygoill
 -- # Last Change:  2022 Jan 31
```

Comment:
It's repeated

Reply:
And this file is generated, so I think it should be regenerated from upstream file instead...

Reply:
I noticed, but I have no idea how to regenerate it. I could give it a go, I guess? :innocent:

Reply:
Good question. Why is this not documented in `MAINTAIN.md`?

Reply:
(although in this case I think we can safely skip the regeneration, as we only touch the comment header.)

Reply:
It also looks like vim9jit was updated slightly since the files were generated, so I might have to rerun it for the others too. I think it's best I leave it for now :+1:

Reply:
I don't know how to use `vim9jit` either

Reply:
@tjdevries

Reply:
Oh, actually this is the only vim9jitted file? Regardless, I'll leave it for now. :innocent:

Reply:
No, `cfilter` is, too (but not included in this patch).